lib/server.rb

```diff
@@ -1,10 +1,67 @@
-require "sinatra"
+require "grape"
+require "grape-swagger"
 
-class Server < Sinatra::Base
-  get "/" do
-    repo = Repositories::ExampleRepository.new
-    res = repo.test_query
+class Server < Grape::API
+  version "v1", using: :path, vendor: "forms"
+  format :json
+  prefix :api
 
-    Services::Example.new.execute(res[:result])
+  resource :forms do
+    desc "Create a form."
+    params do
+      requires :form, type: String, desc: "Form data."
+    end
+    post do
+      {
+        form_id: 1,
+        form_name: "form name",
+        user: "user@example.com"
```

Comment:
i'd probably go with:

- `id` (the form is redundant)
- `name` (same)
- `submission_email` - it'll be where we send the submissions

Reply:
Nice, yeah that makes sense